go/models/document.go

```diff
@@ -0,0 +1,14 @@
+package models
+
+import (
+	"gorm.io/gorm"
+)
+
+type Document struct {
+	gorm.Model
+	Title       string `gorm:"not null"`
+	Description string `gorm:"not null"`
```

Comment:
```suggestion
	Description string
```

Je suis pas sur pour le not null sur la description ?

Je pense pas qu'avoir une description sur un passport serve à grand chose par exemple